cmd/fake-nvidia/main.go

```
@@ -0,0 +1,186 @@
+	if len(args) == 0 {
+		usage(stderr)
+		return errors.New("command is required")
+	}
+	catalog, err := profiles.LoadCatalog()
+	if err != nil {
+		return err
+	}
+	switch args[0] {
+	case "profiles":
+		for _, id := range catalog.ProfileIDs() {
+			fmt.Fprintln(stdout, id)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)